tap_facebook/client.py

```diff
@@ -26,8 +26,8 @@ class FacebookStream(RESTStream):
 
     @property
     def url_base(self) -> str:
-        version = self.config.get("api_version", "")
-        account_id = self.config.get("account_id", "")
+        version = self.config.get("api_version")
+        account_id = self.config.get("account_id")
```

Comment:
We should probably document this but having a `default` in the setting makes our validation step add it to the config dict if it's not present. `account_id` is always present because it's required.

```suggestion
        version: str = self.config["api_version"]
        account_id: str = self.config["account_id"]
```

Reply:
https://github.com/meltano/sdk/issues/1727